### PR TITLE
CDAP-9178 logs for spark programs

### PR DESCRIPTION
**Pull request by deepakwadhwani**
- Move unnecessary loggers to WARN. 
- Fix mapreduce error.

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java [hunk 1]
```diff
@@ -389,7 +389,8 @@ protected void run() throws Exception {
     // Shutdown will still get executed, but the service will notify failure after that.
     // However, if it's the job is requested to stop (via triggerShutdown, meaning it's a user action), don't throw
     if (!stopRequested) {
-      Preconditions.checkState(job.isSuccessful(), "MapReduce JobId {} failed", job.getStatus().getJobID());
+      Preconditions.checkState(job.isSuccessful(), String.format("MapReduce JobId %s failed",
```
Comment (chtyim) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java]: Preconditions supports `%s` already, no need to use `String.format`.

Comment (deepakwadhwani) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java]: @chtyim fixed it, can you please take another look.

**Comment by chtyim** [comment]
LGTM

**Comment by deepakwadhwani** [comment]
Clean build: https://builds.cask.co/browse/CDAP-RUT991-5

**Pull request merged**